twty.go

```diff
@@ -455,6 +457,31 @@ func main() {
 				showTweets(tweets[:], *verbose)
 			}
 		}
+	} else if len(*fromfile) > 0 {
+		var text []byte
+		if *fromfile == "-" {
+			text, err = ioutil.ReadAll(os.Stdin)
+			if err != nil {
+				log.Fatal("failed to read a new tweet from Stdin:", err)
+			}
+		} else {
+			var fd *os.File
+			fd, err = os.Open(*fromfile)
+			if err != nil {
+				log.Fatal("failed to open:", err)
+			}
+			text, err = ioutil.ReadAll(fd)
+			fd.Close()
```

Comment:
use ioutil.ReadFile instead. And make new func `readFile(*fromfile)` both read stdin or the file pointed to `*fromfile`.